pykickstart/commands/btrfs.py

```diff
@@ -126,14 +126,45 @@ def level_cb(value):
             else:
                 raise KickstartParseError(formatErrorMsg(self.lineno, msg=_("Invalid btrfs level: %s") % value))
 
-        op = KSOptionParser()
-        op.add_argument("--noformat", dest="format", action="store_false", default=True)
-        op.add_argument("--useexisting", dest="preexist", action="store_true", default=False)
+        op = KSOptionParser(prog="btrfs", description="""
+.. versionadded:: Fedora17
+
+Defines a BTRFS volume or subvolume. This command is of the form:
+
+``btrfs <mntpoint> --data=<level> --metadata=<level> --label=<label> <partitions*>``
+
+for volumes and of the form:
+
+``btrfs <mntpoint> --subvol --name=<path> <parent>``
+
+for subvolumes.
+
+The ``<partitions*>`` (which denotes that multiple partitions can be
+listed) lists the BTRFS identifiers to add to the BTRFS volume. For
+subvolumes, should be the identifier of the subvolume's parent volume.
+
+``<mntpoint>``
+
+    Location where the file system is mounted.""")
+        op.add_argument("--noformat", dest="format", action="store_false",
+                        default=True, help="""
+    Use an existing BTRFS volume (or subvolume) and do not reformat the
+    filesystem.""")
+        op.add_argument("--useexisting", dest="preexist", action="store_true",
```

Comment:
Not having this indented makes the source fairly ugly. It's simple to handle though:

`from textwrap import dedent`

and then wrap the multiline strings in a dedent() call. I'd indent them over to the level of the first arg of KSOptionParser, but not further.